### PR TITLE
Clean up upnp YAML config

### PR DESCRIPTION
**Pull request by StevenLooman**
## Proposed change

Remove `local_ip` configuration option as this is no longer supported.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/55540
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by StevenLooman** [comment]
Change of plans.

**Comment by MartinHjelmare** [comment]
It's ok to remove the option from the docs also if we deprecate the option.

**Comment by StevenLooman** [comment]
Uncertain why the three top items haven't been executed by the automated build. Perhaps these are only executed after the require review has been done?

File: source/_integrations/upnp.markdown [hunk 1]
```diff
@@ -36,21 +36,3 @@ Alternatively, you can use YAML by adding the following section to your `configu
 # Example configuration.yaml entry
 upnp:
 ```
```
Comment (bdraco) [inline source/_integrations/upnp.markdown:38]:
```suggestion
```
```

**Comment by bdraco** [comment]
Looks like the CI failed.  I did an same suggestion empty commit to trigger a retry

File: source/_integrations/upnp.markdown [hunk 2]
```diff
@@ -35,22 +35,3 @@ Alternatively, you can use YAML by adding the following section to your `configu
 ```yaml
 # Example configuration.yaml entry
 upnp:
-```
```
Comment (klaasnicolaas) [inline source/_integrations/upnp.markdown:38]: This should be put back (```)

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @StevenLooman 👍

**Comment by StevenLooman** [comment]
Thank you @frenck!

**Pull request merged**